src/index.js

```diff
@@ -1,32 +1,43 @@
 var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
 
-var API = 'https://rickandmortyapi.com/api/character/';
+var API = "https://rickandmortyapi.com/api/character/";
```

Comment:
Debemos de utilizar la palabra reservada const en vez de var. Estándar de ECMAScript 6.
```suggestion
const API = "https://rickandmortyapi.com/api/character/";
```